CoastalResilienceNetwork/GeositeFramework
# Translation Follow Ups

* Fixes Save & Share title translation
* Fixes Measure tooltip and results bubble
* Fixes checkbox and continue button on InfoGraphic
* Sample spanish translation for region and measure plugin.

Connects #606 

Test:
* Change your region language to "es"
* See the following are translated:
    * See Save & Share title, 
    * Measure tooltip, in-progress template and final result
    * Identify Point infographic built-in text

![screenshot from 2016-03-28 16 15 55](https://cloud.githubusercontent.com/assets/1014341/14088988/81af778e-f500-11e5-865d-0741a0596918.png)
![screenshot from 2016-03-28 15 17 33](https://cloud.githubusercontent.com/assets/1014341/14088989/81b9a70e-f500-11e5-8ce3-62f97a9cbcba.png)
